crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -2042,7 +2042,7 @@ pub async fn handle_resolve_error(
                 request_type: format!("{} request", reference_type.into_value()),
                 request,
                 resolve_options,
-                error_message: Some(err.to_string()),
+                error_message: Some(format!("{:?}", err)),
```

Comment:
```suggestion
                error_message: Some(format!("{:?}", PrettyPrintError(&err))),
```

Reply:
true, I guess I can refactor the output of `ResolvingIssues` to use that